### ISSUE TITLE
Using --time switch throws error

### ISSUE THREAD

**Issue by ghost** [Compass/compass#1478]
Running compass with the `--time` throws an error like:

```
NoMethodError on line ["175"] of lib/compass/compiler.rb: undefined method `__duration'
```

Looks like this line:

https://github.com/chriseppstein/compass/blob/stable/lib/compass/compiler.rb#L143

Issue can be recreated simply:

```
$ compass create test
$ cd test
$ compass watch --time
```

Compass 1.0.0.alpha.16
Sass 3.3.0.rc.2 (Maptastic Maple)

**Comment by mgol** [comment]
Just got hit by it, especially that the `grunt-contrib-compass` Grunt plugin uses the `--time` option with no way of disabling it.

**Comment by RaphaelDDL** [comment]
Same as mzgol, got hit using `grunt-contrib-compass` also .

**Comment by chriseppstein** [comment]
fixed in `1.0.0.alpha.17`
